ethcore/src/engines/validator_set/simple_list.rs

```diff
@@ -33,9 +33,14 @@ pub struct SimpleList {
 impl SimpleList {
 	/// Create a new `SimpleList`.
 	pub fn new(validators: Vec<Address>) -> Self {
-		SimpleList {
-			validators: validators,
+		let validator_count = validators.len();
+		if validator_count == 1 {
+			warn!(target: "engine", "Running AuRa with a single validator implies instant finality. Use a database?");
 		}
+		if validator_count % 2 == 0 {
+			warn!(target: "engine", "Running AuRa with an even number of validators is not recommended (risk of network split).");
```

Comment:
Just realised that it might get pretty spammy on misconfigured networks using validator set contract, can we verify that and maybe throttle if that's the case?

Reply:
Hmm, I think we create `SimpleLists` fairly rarely and if they are running with a not-so-great config shouldn't we tell them about it? On `kovan` this message is shown twice when there are 4 validators; is that too spammy?

Reply:
@tomusdrw is your concern here still valid?

Reply:
All good!